### ISSUE TITLE
Add logging configuration option for rserver

### ISSUE THREAD

**Issue by colearendt** [rstudio/rstudio#3308]
Affects RStudio Server & RStudio Server Pro

### Describe the problem in detail

On occasion (i.e. in docker containers), it is ideal to be able to execute the `rserver` process directly without a supervisor, system daemon, etc.  In this context, it is ideal that programs send output to `stdout` and `stderr`.

Today, there is no ability to configure `rserver` to send output to stdout or configure the logging level.  As a result, rserver runs silently (except for `stderr`) and presents problems for debugging, server status checking, etc.

The rocker project (https://github.com/rocker-org/rocker-versioned/blob/master/rstudio/Dockerfile) gets around this issue by using the [s6 init system](https://skarnet.org/software/s6/), which seems to send the typical syslog to stdout.  Other, more lightweight init systems (or directly executing the binary) require giving up more granular logging of the rserver process.

I imagine something like:

```
/usr/lib/rstudio-server/bin/rserver --server-daemonize=0 --log-output=/dev/stdout --log-level=DEBUG
```

**Comment by tblazina** [comment]
hi @colearendt, do you happen to have figured out a solution to this? I'm actually running the rocker project rstudio version you mention however the s6 init system doesn't seem to resolve the issue of Rstudio not sending output to `stdout` or `stderr`. I've created an [issue](https://github.com/rocker-org/rocker/issues/361#issuecomment-535581352) there but I have yet to figure out how to actually get this to work even with the s6 init system.

**Comment by jmcphers** [comment]
We implemented this recently in RStudio 1.3. You can read some pre-release documentation here:

https://docs.rstudio.com/ide/server-pro/1.3.465-2/server-management.html#logging
